### PR TITLE
Quick settings cancel tap

### PR DESCRIPTION
**Pull request by jspkay**
I am not sure what is going on with all these changes. The only actual change to consider is the one to QuickSettings.cpp.
I am sorry for the inconvenience, I'm learning git practicing.

Anywas, the pr proposes to cancel a tap when the finger is dragged onto other elements

### CODE REVIEW

**Comment by minacode** [comment]
This is sadly not mergeable in this state. You have to create a clean PR, because the maintainers can only accept the whole PR or nothing in the end.

File: src/displayapp/screens/Notifications.cpp [hunk 1]
```diff
@@ -135,7 +135,7 @@ bool Notifications::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
     }
     return false;
   }
-
+
```
Comment (minacode) [inline src/displayapp/screens/Notifications.cpp:138]: This is not needed.

File: src/displayapp/screens/settings/QuickSettings.cpp [hunk 2]
```diff
@@ -10,8 +10,12 @@ using namespace Pinetime::Applications::Screens;
 namespace {
   void ButtonEventHandler(lv_obj_t* obj, lv_event_t event) {
     auto* screen = static_cast<QuickSettings*>(obj->user_data);
-    if (event == LV_EVENT_CLICKED) {
-      screen->OnButtonEvent(obj);
+    static lv_obj_t* pressedObj = nullptr;
```
Comment (minacode) [inline src/displayapp/screens/settings/QuickSettings.cpp:13]: I have a bad feeling with introducing a local static variable, but that might just be me. Is there any other way to implement this? I am just asking and would like your opinion.

**Review by minacode — COMMENTED** [review]
Please also fix the missing spaces in your code. You can run clang-format with the file found [here](https://docs.infinitime.io/en/latest/how-to-contribute.html?highlight=Format#tl-dr)

File: src/displayapp/screens/settings/QuickSettings.cpp (hunk 2 above)
Comment (jspkay) [reply src/displayapp/screens/settings/QuickSettings.cpp:13]: I think it's not a problem. The compiler should simply allocate some space near the function itself. I am not 100% sure tough. We should ask some experienced programmer. 
Regarding the implementation, there surely are other ways to implement this. I can think of a global variable (although scoped) but I think it's just worse, otherwise it's possible to add another member to the class, but it would be unnecessary, as I see this.
In conclusion, I guess a static variable is the simplest and safest way of doing this, tbh

Comment (minacode) [reply src/displayapp/screens/settings/QuickSettings.cpp:13]: Ok, I agree with you.

**Comment by Riksu9000** [comment]
I like the idea. I think the dragging behavior should be changed, however doing it this way would require fixing it in each screen individually. We're preparing to use our own fork of LVGL, so we can fix issues like this at the root of the problem. If you're interested, I encourage you to look into fixing this issue directly in LVGL here https://github.com/InfiniTimeOrg/lvgl
